src/components/gauge/readme.md

```diff
@@ -0,0 +1,37 @@
+# gx-gauge
+
+This component allows you to display information in the form of ranges, and decide whether you want to show it using linear or circular gauges.
+
+Use `gx-gauge-range` element to set the number of ranges and the max value the gauge will have.
+
```

Comment:
Adding a sample would be really nice.

Reply:
Sample added in commit 3068cfc8c6082ae066003baabf10cc969bd375b7